apis/teamspeak_api.php

```diff
@@ -370,6 +370,78 @@ public function stopServer($sid)
         return (object) $result;
     }
 
+    /**
+     * Suspends an existing virtual server
+     *
+     * @param int $sid The virtual server ID
+     * @return stdClass An object containing the request response
+     */
+    public function suspendServer($sid)
+    {
+        try {
+            // Build the parameters array
+            $api_params = [
+                'virtualserver_autostart' => 0
+            ];
+
+            // Fetch the virtual server instance, and modify it
+            $instance = $this->apiRequest('serverquery')->serverList()[$sid];
```

Comment:
I'd prefer we use $this->getServer($sid);

Reply:
Done.